tests/unit/mocks/jobsMockData.ts

```diff
@@ -24,8 +24,8 @@ const partData = [
     description: 'string',
     resolutionMeter: 8000,
     resolutionDegree: 0.703125,
-    imagingTimeEndUTC: new Date('2024-01-28T13:47:43.427Z'),
-    imagingTimeBeginUTC: new Date('2024-01-28T13:47:43.427Z'),
+    imagingTimeEndUTC: '2024-01-28T13:47:43.427Z',
```

Comment:
Consider handle the date type and define the relevant type for each of those mocks data

Reply:
done